### PR TITLE
Feature/flho 321 multiple forms refactor

### PR DESCRIPTION
**Pull request by bendiggle**
* Created shooting-clubs directory
* Created common directory
* Moved activity field and translations and acceptance steps to common directory,
* Removed from new-dealer directory,
* Changed app.js to use common direcotry and shooting-clubs. Also changed it so will look at common directory for some acceptance tests when node_env = ci

### CODE REVIEW

**Comment by easternbloc** [comment]
when this is merged please squash and merge so we get rid of that `remove README` commit

File: apps/shooting-clubs/fields/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (joefitter) [inline apps/shooting-clubs/fields/index.js]: this needed?

Comment (easternbloc) [inline apps/shooting-clubs/fields/index.js]: lol

Comment (bendiggle) [inline apps/shooting-clubs/fields/index.js]: @joefitter Once this is merged, there will be another PR with this file having stuff init!

Comment (easternbloc) [inline apps/shooting-clubs/fields/index.js]: nothing better than a file with stuff in it tbh
